app/src/main/java/com/github/palFinderTeam/palfinder/ui/login/LoginActivity.kt

```diff
@@ -72,7 +72,7 @@ class LoginActivity : AppCompatActivity() {
 
     // onCreate One Tap version
     override fun onCreate(savedInstanceState: Bundle?) {
-        val sharedPref = getPreferences(Context.MODE_PRIVATE) ?: return
+        val sharedPref = getSharedPreferences("theme",Context.MODE_PRIVATE) ?: return
```

Comment:
Some hardcoded string

Reply:
Indeed! this will be changed during the final checks for constants